### ISSUE TITLE
Failed to install python wrapper on MacOS (python 3)

### ISSUE THREAD

**Issue by avostryakov** [google/sentencepiece#82]
> pip install sentencepiece
Collecting sentencepiece
  Using cached https://files.pythonhosted.org/packages/ef/ba/17c0c4f8ccc746b2182c7e3c8292be0bdb37fbadeaf467d2f69565160764/sentencepiece-0.0.7.tar.gz
Building wheels for collected packages: sentencepiece
  Running setup.py bdist_wheel for sentencepiece ... error
  Complete output from command /Users/vostryakov/projects/3env/bin/python -u -c "import setuptools, tokenize;__file__='/private/var/folders/s4/l8hg1ch969d9p96z4bwfllmc0000gp/T/pip-install-gni78bgz/sentencepiece/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /private/var/folders/s4/l8hg1ch969d9p96z4bwfllmc0000gp/T/pip-wheel-60ftqmg6 --python-tag cp36:
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.macosx-10.11-x86_64-3.6
  copying sentencepiece.py -> build/lib.macosx-10.11-x86_64-3.6
  running build_ext
  building '_sentencepiece' extension
  creating build/temp.macosx-10.11-x86_64-3.6
  clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/include/python3.6m -c sentencepiece_wrap.cxx -o build/temp.macosx-10.11-x86_64-3.6/sentencepiece_wrap.o -std=c++11 -g -O2 -I/usr/local/include
  sentencepiece_wrap.cxx:3123:10: fatal error: 'sentencepiece_trainer.h' file not found
  #include <sentencepiece_trainer.h>
           ^
  1 error generated.
  error: command 'clang' failed with exit status 1
  
  ----------------------------------------
  Failed building wheel for sentencepiece

The same error when I try to build from a source.

**Comment by lemontheme** [comment]
Can report that I'm having the same problem, although pip seems to get farther in OP's case than it does in mine, so perhaps it's unrelated. This is what the output currently looks like:

```
Collecting sentencepiece
  Using cached https://files.pythonhosted.org/packages/ef/ba/17c0c4f8ccc746b2182c7e3c8292be0bdb37fbadeaf467d2f69565160764/sentencepiece-0.0.7.tar.gz
    Complete output from command python setup.py egg_info:
    Package sentencepiece was not found in the pkg-config search path.
    Perhaps you should add the directory containing `sentencepiece.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'sentencepiece' found
    Failed to find sentencepiece pkgconfig

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/1p/5zq7ks1d0tvgp15f25kfc8m80000gn/T/pip-install-3n809bmg/sentencepiece/
```

**Comment by lemontheme** [comment]
Yup, that's how I did it in the end. May I suggest adding something to the README that says as much?

**Comment by taku910** [comment]
Thank you for the suggestion.
Updated the python/README.md
https://github.com/google/sentencepiece/tree/master/python#build-and-install-sentencepiece

**Comment by avostryakov** [comment]
Thank you! I didn't install SentencePiece C++ before "python setup.py build". it works correctly
